src/main/java/frc/robot/RobotContainer.java

```diff
@@ -56,9 +86,28 @@ public RobotContainer() {
    */
   private void configureButtonBindings() {
     /* Driver Buttons */
-    zeroGyro.whenPressed(new InstantCommand(() -> s_Swerve.zeroGyro()));
+    zeroGyro.onTrue(new InstantCommand(() -> s_Swerve.zeroGyro()));
+    xButton.onTrue(new InstantCommand(() -> toggleGamePiece()));
+    gripButton.whileTrue(new GripObject(s_Gripper, isCone, true));
+    gripButtonReverse.whileTrue(new GripObject(s_Gripper, isCone, false));
+
+    /* D-Pad Controller Input Detection */
+    dpadUp.onTrue(new RotateToHeading(s_Swerve, 0));
+    dpadRight.onTrue(new RotateToHeading(s_Swerve, 90));
+    dpadDown.onTrue(new RotateToHeading(s_Swerve, 180));
+    dpadLeft.onTrue(new RotateToHeading(s_Swerve, 270));
+
+    dpadUp.onTrue(new InstantCommand(() -> dPadPOV(0)));
+    dpadRight.onTrue(new InstantCommand(() -> dPadPOV(90)));
+    dpadDown.onTrue(new InstantCommand(() -> dPadPOV(180)));
+    dpadLeft.onTrue(new InstantCommand(() -> dPadPOV(270)));
```

Comment:
If you change the dPadPOV method so it returns a new RotateToHeading object, you can consolidate the 8 lines above into 4. That way we only have one "onTrue" call for each button.

Reply:
Done! Called new RotateToHeading Object in dPadPOV()